### PR TITLE
Fixed #25932 -- Made predicates of OGRGeometry return bool instead of int.

### PR DESCRIPTION
**Pull request by sir-sigurd**
https://code.djangoproject.com/ticket/25932

### CODE REVIEW

File: tests/gis_tests/gdal_tests/test_geom.py [hunk 1]
```diff
@@ -493,3 +493,42 @@ def test_equivalence_regression(self):
         "Testing equivalence methods with non-OGRGeometry instances."
         self.assertIsNotNone(OGRGeometry('POINT(0 0)'))
         self.assertNotEqual(OGRGeometry('LINESTRING(0 0, 1 1)'), 3)
+
+    def test_contains(self):
+        self.assertIs(OGRGeometry('POINT(0 0)').contains(OGRGeometry('POINT(0 0)')), True)
```
Comment (claudep) [inline tests/gis_tests/gdal_tests/test_geom.py]: I think we generally use `assertTrue()` instead of `assertIs(..., True)` in Django.

Comment (charettes) [inline tests/gis_tests/gdal_tests/test_geom.py]: I think `assertIs` was used because `assertTrue(1)` doesn't fail.

Comment (claudep) [inline tests/gis_tests/gdal_tests/test_geom.py]: Ah, right! Then a comment might be nice to shut up dumb reviewers like me :-)

Comment (sir-sigurd) [inline tests/gis_tests/gdal_tests/test_geom.py]: This makes sense, added a comment.

**Comment by timgraham** [comment]
merged in a6c803a2e3d270818d20f3f0c76e2241de9f0ab1, thanks!

**Pull request closed without merging**